### PR TITLE
solved question 1 working on #2

### PR DESCRIPTION
**Pull request by AndresBallares**

### CODE REVIEW

File: recipe.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Healthy Protein Pancakes</title>
+</head>
+<body>
+    <h2><Healthy Protein Pancake/h2>
```
Comment (Angiepmunoz) [inline recipe.html]: This should be the main header, which is a specific header number.

File: recipe.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    <img
+    src= "https://www.erinliveswhole.com/wp-content/uploads/2021/04/proteinpancakes-5.jpg"
+    height = "300px"
+    />
+    <div>
+        ---------------------------------------------------------------------------------------------------------------------------------------------------
```
Comment (Angiepmunoz) [inline recipe.html]: there is an element for this

File: recipe.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Healthy Protein Pancakes</title>
+</head>
+<body>
+    <h2><Healthy Protein Pancake/h2>
+    <img
+    src= "https://www.erinliveswhole.com/wp-content/uploads/2021/04/proteinpancakes-5.jpg"
```
Comment (Angiepmunoz) [inline recipe.html]: This is missing a specified attribute, which is why it's failing. Refer to the test specs.

File: recipe.html (hunk 1 above)
Comment (AndresBallares) [reply recipe.html]: solved

File: recipe.html (hunk 2 above)
Comment (AndresBallares) [reply recipe.html]: Angie I used the <hr> but the test still have not been able to adjust to length of 4

File: recipe.html (hunk 3 above)
Comment (AndresBallares) [reply recipe.html]: solved.  thanks

Comment (AndresBallares) [reply recipe.html]: also added the alt="Protein Pancakes!"

**Pull request closed without merging**